trunner/device.py

```diff
@@ -441,7 +441,7 @@ def reboot(self, serial_downloader=False, cut_power=False):
             self._restart_by_jtag()
 
     def flash(self):
-        self.reboot(serial_downloader=True, cut_power=True)
+        self.reboot(serial_downloader=True, cut_power=False)
```

Comment:
maybe instead of changing the parameters in multiple places, we could use instance attribute and change the parameters / code flow accordingly?